src/Workspaces/Core/Portable/Utilities/EventMap.cs

```diff
@@ -154,9 +157,9 @@ internal struct EventHandlerSet<TEventHandler>
         {
             private ImmutableArray<Registry<TEventHandler>> _registries;
 
-            internal EventHandlerSet(object registries)
```

Comment:
📝 This was typed to `object` with a boxing allocation because `EventHandlerSet<TEventHandler>` was internal while `Registry<TEventHandler>` was private.